code/pizzabot.js

```diff
@@ -1,7 +1,86 @@
+
+// Pizza sorts:
+
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
```

Comment:
If you didn't want your user to type "Vegetarian Pizza", and maybe just "Vegetarian".  You could update these values